app/views/my_facilities/blood_pressure_control.html.erb

```diff
@@ -152,9 +152,8 @@
       <ol>
         <li><strong>Overall BP control:</strong>
           <ul>
-            <li>Numerator: Patients registered at this facility whose most recent BP in the past 90 days was &lt;140/90.</li>
-            <li>Denominator: Patients registered at this facility.</li>
-            <li>Patients registered in the last 2 months are excluded from both the <i>Numerator</i> and <i>Denominator</i>.</li>
+            <li>Numerator: Patients whose last BP in the past 90 days was &lt;140/90.</li>
+            <li>Denominator: Patients with this preferred facility.</li>
```

Comment:
Can we use the actual term we are refactoring towards here? "Patients with this assigned facility" ?  Thats what the queries are changing to use, right?

Having multiple terms for the same concept adds confusing for us and for our users when we do training or help docs and the like.